src/desktop/src/ui/views/exchanges/MoonPay/AddAmount.js

```diff
@@ -539,4 +543,7 @@ const mapDispatchToProps = {
     setDenomination,
 };
 
-export default connect(mapStateToProps, mapDispatchToProps)(withTranslation()(AddAmount));
+export default connect(
+    mapStateToProps,
+    mapDispatchToProps,
+)(withTranslation()(AddAmount));
```

Comment:
@rihardsgravis are your versions of `eslint` and `prettier` up to date? The latest versions should be putting this on a single line instead of multiple